backend-api/src/functions/adapters/kmsAdapter.ts

```diff
@@ -7,16 +7,14 @@ import { errorResult, Result, successResult } from "../utils/result";
 import { NodeHttpHandler } from "@smithy/node-http-handler";
 
 export class KMSAdapter implements IKmsAdapter {
-  private readonly kmsClient = new KMSClient([
-    {
-      region: "eu-west-2",
-      maxAttempts: 2,
-      requestHandler: new NodeHttpHandler({
-        connectionTimeout: 29000,
-        requestTimeout: 29000,
-      }),
-    },
-  ]);
+  private readonly kmsClient = new KMSClient({
+    region: "eu-west-2",
+    maxAttempts: 2,
+    requestHandler: new NodeHttpHandler({
```

Comment:
I think we need to have a think about this! The lambda timeout is under this threshold, so we'll never hit this 29000 value

Reply:
I will revert this change and fix it in another PR